src/WebJobs.Script.WebHost/Helpers/VfsSpecialFolders.cs

```diff
@@ -76,7 +76,7 @@ public static IEnumerable<VfsStatEntry> GetEntries(string baseAddress, string qu
                         MTime = dir.LastWriteTimeUtc,
                         CRTime = dir.CreationTimeUtc,
                         Mime = "inode/shortcut",
-                        Href = baseAddress + Uri.EscapeUriString(SystemDriveFolder + VirtualFileSystem.UriSegmentSeparator) + query,
+                        Href = baseAddress + Uri.EscapeDataString(SystemDriveFolder + VirtualFileSystem.UriSegmentSeparator) + query,
```

Comment:
Obsolete in .NET 6 -- https://docs.microsoft.com/en-us/dotnet/api/system.uri.escapeuristring?view=net-6.0